### ISSUE TITLE
v2.2.8 I can stretch the UI wider, but can't bring it back to smaller

### ISSUE THREAD

**Issue by davidc86** [sillsdev/ptx2pdf#710]
v2.2.8 I can stretch the UI wider, but I can't bring it back to smaller.
To restore the UI, I have to exit PTXprint (thankfully). 

Though this does mean that those who want a different sized window, will have to resize their window larger each time they restart PTXprint, correct?

(I think resetting the size on a restart is a good feature, but there might be others who get stressed by it.)

**Comment by markpenny** [comment]
Yes, this has always been a bit of an issue. I'm not sure how we can get away from it. The problem is that the picture fills the available space (either vertically or horizontally) and while the picture 'holds' that space, we can't shrink the window.

A convenient (but not very discoverable) workaround is to select the Position option which shows a smaller thumbnail of the relative location on the page:

![image](https://user-images.githubusercontent.com/12962708/182484740-b34ae85b-4b2c-4f13-afc4-99ef8901ad70.png)

With this smaller image showing you *can* actually resize the window, but usually it takes several steps to do so (pull up the bottom, then pull in the side, and repeat if needed). Far from ideal, but easier than restarting PTXprint.

I'll tag this as an enhancement request and keep thinking of ways to solve it; but don't hold your breath!

**Comment by markpenny** [comment]
Is this more of a request to remember the UI window size between sessions?

**Comment by davidg-sil** [comment]
I think it's a usability request. Typically if you make a window bigger, you can shrink it again! I'm guessing there is  some glade/gtk flag somewhere that tells it that graphics can redraw to be smaller, which isn't set.
